### ISSUE TITLE
Opening URL in private window

### ISSUE THREAD

**Issue by RichardMore** [carnager/rofi-pass#62]
If I want to open an URL from command line e.g. w/ firefox the command is `firefox -private-window mysite.com` but if in the pass entry when I use `url: -private-window mysite.com` it will only open a private window w/o the URL pasted and submitted. Using quotation mark does not help.

**Comment by carnager** [comment]
this should rather be a switch, I might add this

**Comment by RichardMore** [comment]
Yes, a switch would be great, but you need to consider that most browsers have different command for private browsing window, eg. firefox has **-private-window**, while google-chrome has **-incognito**.
Without a switch I achieved this by modifying the line:
`$BROWSER "$(pass "$selected_password" | grep "${URL_field}: " | gawk '{sub(/:/,"")}{print $2}1' | head -1)"; exit;`
into the following:
`GOTURL=$(pass "$selected_password" | grep "${URL_field}: " | sed "s/${URL_field}: //g")
  $BROWSER $GOTURL`
at the **openURL** method.

**Comment by moviuro** [comment]
I'd use a new line in the pass file (say: `^private$` or `^incognito$`); if that line with a magic operator exists, then set $incognito_switch for the current $BROWSER (or hard fail in case $BROWSER is not set); then call `$BROWSER $incognito_switch $url`
